src/views/EnemyStatus.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { get, keys, map } from "lodash";
+import { Box, FormControl, makeStyles, TextField } from "@material-ui/core";
+import { STAT_FIELDS } from "../constants/EnemyStats";
+import ELEMENTS from "../constants/Element";
+import { ToggleButton, ToggleButtonGroup } from "@mui/material";
+
+const useStyles = makeStyles({
+  field: {
+    "&:hover .MuiInputLabel-root": {
+      color: "#90caf9", //Change hover color of text
+    },
+    "& .Mui-focused": {
+      color: "#90caf9", //Color of label when focused
+    },
+  },
+  elementSelect: {
+    display: "flex",
+    alignItems: 'center'
+  },
+  elementIcon: {
+    height: "30px",
+    width: "auto",
+    display: "flex",
+  },
+});
+
+function EnemyStatus(props) {
+  const classes = useStyles();
+  const [element, setElement] = useState("None");
+
+  return (
+    <Box>
+      <>
+        {map(STAT_FIELDS, (stat, idx) => {
+          return (
+            <Box
+              key={`${stat}${idx}`}
```

Comment:
i don't think you should use the object 'stat' as part of the react key, should be a string / number